CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Forge
+
+#### Changed
+
+- You can now pass arguments to `cairo-profiler` or `cairo-coverage`,
+  e.g. `snforge test --coverage --include tests-functions`. If flags name duplicates a flag in `snforge test`, you can
```

Comment:
This sentence doesn't really sound clear to me. Is `snforge test --coverage --include tests-functions` proper or not?